python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -0,0 +1,781 @@
+        default=False,
+    )
+    usePsf = pexConfig.Field(
+        dtype=bool,
+        doc="Option to use the PSF as part of the convolution; requires `useFFT=True`.",
+        default=False,
+    )
+    dcrNSubbands = pexConfig.Field(
+        dtype=int,
+        doc="Number of sub-bands to forward model chromatic effects to fit the supplied exposures.",
+        default=3,
+    )
+    maxNIter = pexConfig.Field(
+        dtype=int,
+        doc="Maximum number of iterations of forward modeling.",
+        default=8,
+    )
+    minNIter = pexConfig.Field(
+        dtype=int,
+        doc="Minimum number of iterations of forward modeling.",
+        default=3,
+    )
+    convergenceThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Target change in convergence between iteration of forward modeling.",
+        default=0.001,
+    )
+    useConvergence = pexConfig.Field(
+        dtype=bool,
+        doc="Turn on or off the convergence test as a forward modeling end condition.",
+        default=True,
+    )
+    doWeightGain = pexConfig.Field(
+        dtype=bool,
+        doc="Use the calculated convergence metric to accelerate forward modeling.",
+        default=True,
+    )
+    doAirmassWeight = pexConfig.Field(
+        dtype=bool,
+        doc="Weight exposures by airmass.",
+        default=True,
+    )
+    clampModel = pexConfig.Field(
+        dtype=float,
+        doc="Restrict new solutions from changing by more than a factor of `clampModel`.",
+        default=2.,
+    )
+    regularizeSigma = pexConfig.Field(
+        dtype=float,
+        doc="Set the threshold to exclude noise-like pixels from frequency regularization.",
+        default=3.,
+    )
+    clampFrequency = pexConfig.Field(
+        dtype=float,
+        doc="Restrict solutions from changing by more than a factor of `clampModel` between frequencies.",
+        default=2.,
+    )
+    useNonNegative = pexConfig.Field(
+        dtype=bool,
+        doc="Force the model to be non-negative.",
+        default=False,
+    )
+    maxGain = pexConfig.Field(
+        dtype=float,
+        doc="Maximum convergence-weighted gain to apply between forward modeling iterations.",
+        default=2.,
+    )
+    minGain = pexConfig.Field(
+        dtype=float,
+        doc="Minimum convergence-weighted gain to apply between forward modeling iterations.",
+        default=0.5,
+    )
+    convergenceMaskPlanes = pexConfig.ListField(
+        dtype=str,
+        default=["DETECTED"],
+        doc="Mask planes to use to calculate convergence."
+    )
+    lambdaEff = pexConfig.Field(
+        dtype=float,
+        doc="Effective center wavelength of the filter, in nm. This will be replaced in DM-13668.",
+        default=478.,
+    )
+    filterWidth = pexConfig.Field(
+        dtype=float,
+        doc="Width of the filter, in nm. This will be replaced in DM-13668",
+        default=147.,
+    )
```

Comment:
I don't think there should be any hard-wired filter information in the `Config`. You should get this from the `Filter` object, which you can get using the `filter` from the data identifier. This doesn't need the full transmission curves (DM-13668).